### ISSUE TITLE
[bench request]: Add Compass 0.2.0 (Qwen3.5-4B + LoRA, fused verification and option-symbol readouts, TypeSafe wire format)

### ISSUE THREAD

**Issue by adimyth** [fstandhartinger/jevbench#53]
Request to add **Compass 0.2.0** to the ranked systems.

**What it is.** A decision model over a frozen [Qwen/Qwen3.5-4B](https://huggingface.co/Qwen/Qwen3.5-4B) (Apache-2.0, revision `851bf6e8`) plus a LoRA trained through the readouts below (no JevBench data; 8-gram overlap check on every file), with its own readout: each allowed answer is stated as a proposition and verified against the document in its own branch ("is this proposed answer correct under the rubric?"), and that verification log-odds is fused in log space (equal weight) with a direct read of the answer symbols; the weight was chosen on our own internal items. The state is read once and the cache is forked per question and per candidate, so option order cannot reach the model and the reported input tokens are the state plus one rubric plus ~25 tokens per option. No answer generation: every probability is read from the model's logits at one position per branch. Per-type temperatures were fitted on our own items (`dev/`), none of them from JevBench. Code, prompt, calibration and data: https://github.com/adimyth/compass, tag `v0.2.0-submission`, Apache-2.0.

It serves TypeSafe's wire format, so the unchanged `typesafe` adapter works:

```sh
git clone --branch v0.2.0-submission https://github.com/adimyth/compass && cd compass
scripts/serve.sh 8000          # one GPU, bf16, ~9 GB; downloads the pinned weights on first use
# or: docker build -t compass . && docker run --gpus all -p 8000:8000 compass

python -m jevbench.cli run --tasks datasets/public/original.jsonl \
  --adapter typesafe --endpoint http://127.0.0.1:8000 --key-env '' --model compass-0.2.0 \
  --cost-basis self_hosted_gpu --reserve-usd 0 ...
```

`noul` returns `noul`; `choice` and `score` return `probabilities` keyed by option name and level index, computed in float64 and summing to 1. `usage.input_tokens` counts every token the backbone processed.

**Our own numbers on the public items**, one request at a time through your adapter and `score_task`, run once as a final check and not tuned on:

| tier | items | accuracy | well-formed |
| --- | --- | --- | --- |
| easy | 48 | 1.000 | 48/48 |
| standard (original) | 72 | 0.875 | 72/72 |
| hard | 111 | 0.604 | 111/111 |

Hard-tier top-label ECE 0.069; mean TVD to gold distributions on the public `probability` items 0.22. These are public-item figures only and are not comparable to ranked rows, which include the held-out and judge items. Hard misses concentrate in `temporal_numeric` and `long_policy`. We claim no rank from this; the held-out and judge items are yours, and speed and cost are measured from your server.

**Cost basis.** Qwen3.5-4B at the hosted price you use for that size class ($0.03 per M input, DeepInfra), times the input tokens the API reports; nothing is generated. On an RTX 4090 we measured p50 91–174 ms raw (316–1,635 input tokens, verification readout) and 121–145 ms for the release configuration over localhost; your measurement from your server is the one that counts.

**Openness.** Apache-2.0 code; Qwen's weights unchanged (Apache-2.0) plus a 116 MB LoRA adapter we trained on our own generated and model-drafted items (Apache-2.0, `adimyth/compass-lora-v2`, pinned revision), with the training data, splits, shadow suite and provenance in the repository.

If a self-hosted run on your GPU is possible, the command above is all it needs. Happy to answer questions here.

**Comment by fstandhartinger** [comment]
Thanks — this is logged for reachability checks and the JevBench measurement queue. We’ll only make benchmark claims after our own run.

- Harold
